quanto/quantize.py

```diff
@@ -48,6 +50,22 @@ def quantize(model, modules=None, **kwargs):
                 del param
 
 
+def requantize(model, state_dict):
+    # you shouldn't move models that were distributed with accelerate
+    if hasattr(model, "hf_device_map"):
+        raise ValueError(
+            "Model is distributed with accelerate, cannot requantize. Please use an un-distributed model."
+        )
```

Comment:
`quanto` has no dependency to accelerate and should not know anything about that library. If you can deduce something from the model pytorch attributes only (device ?), then you can add a check, otherwise please remove this.